terraform/ecs/config/config.yml

```diff
@@ -0,0 +1,24 @@
+region: eu-west-1
+project: insight
+app_name: explorer
+environment: prod
+# vpc settings
+vpc_id: vpc-09a1702508635ac94
+azs:
+  - eu-west-1a
+  - eu-west-1b
+  - eu-west-1c
+vpc_cidr: 172.31.0.0/16
+public_subnets:
+  - subnet-0124d1b1bca4da686
+  - subnet-0cf08a1f3aa9954a2
+  - subnet-0a963e53f5b50781b
+private_subnets:
+  - subnet-07f6f718172b0c640
+  - subnet-06b67e91b846414f7
+  - subnet-00d01ce070a125e58
+#app insights settings
+image_tag: dev-latest
+max_capacity: 2
```

Comment:
lets put it back to 1 and redeploy the service. Auto scaling can be tricky onindexers.

Reply:
Hey @adamkoy , @Rishikeshpal , this is not an indexer, it's a real time client side visualization tool. It transpiles into static HTML & javascript. All you need is a build pipeline that uploads some files to an S3 website. There is no need for ECS, scaling, or compute for that matter, it doesn't compute anything on the server side.